gpMgmt/test/behave/mgmt_utils/environment.py

```diff
@@ -21,7 +21,8 @@ def before_all(context):
 def before_feature(context, feature):
     # we should be able to run gpexpand without having a cluster initialized
     tags_to_skip = ['gpexpand', 'gpaddmirrors', 'gpstate', 'gpmovemirrors',
-                    'gpconfig', 'gpssh-exkeys', 'gpstop', 'gpinitsystem', 'cross_subnet']
+                    'gpconfig', 'gpssh-exkeys', 'gpstop', 'gpinitsystem', 'cross_subnet',
+                    'gplogfilter']
```

Comment:
@HustonMmmavr, can you explain why we should add new test here?

Reply:
`gplogfilter` behave test works with the mocked log directory, and this directory is filled with files specified by the table of `@given` tag `gplogfeatutre.feature` scenario, thus this test doesn't requires working cluster. 
If we don't add the `gplogfilter` to the `tags_to_skip` array, then:
- `before_feature` function before run test tries to connect to database and would fail with message like:
  ```
  PYTHONPATH=$PYTHONPATH:$PEXPECT_LIB:$TEST_DIR behave $PWD/test/behave/* -s -k --tags=gplogfilter
    HOOK-ERROR in before_feature: InternalError: could not connect to server: Connection refused
        Is the server running on host "localhost" (127.0.0.1) and accepting
        TCP/IP connections on port 6000?
  ```
- `before_scenario/after_scenario` would make a redundant startup of cluster, because this test doesn't require it